storage/pebble/registers/config/config.go

```diff
@@ -0,0 +1,63 @@
+package config
```

Comment:
This is the config for pebble, not for registers, we could move it to storage/pebble/config/config.go